From 75ee19909301377f7b4c1a5c964b618e30b51570 Mon Sep 17 00:00:00 2001
From: Szabolcs Nagy <nsz@port70.net>
Date: Thu, 12 Oct 2023 20:31:48 +0200
Subject: [PATCH] make __getauxval a public ABI symbol

This is needed so that libgcc can access AT_HWCAP without violating
link namespace rules.

Internally musl already used __getauxval symbol for the same reason,
we just remove the hidden marking.
---
 src/include/sys/auxv.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/include/sys/auxv.h b/src/include/sys/auxv.h
index 9358a4a5..63c5bfe9 100644
--- a/src/include/sys/auxv.h
+++ b/src/include/sys/auxv.h
@@ -5,6 +5,6 @@
 
 #include <features.h>
 
-hidden unsigned long __getauxval(unsigned long);
+unsigned long __getauxval(unsigned long);
 
 #endif
-- 
2.34.1

